x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -544,6 +544,19 @@ export const getCatchupTuples = ({
   return catchupTuples;
 };
 
+/**
+ * Takes the rule schedule fields `interval` and `lookback` and uses them to calculate the `from` value for a rule
+ *
+ * @param interval moment.Duration representing the interval on which the rule runs
+ * @param lookback moment.Moment representing the rule's additional lookback
+ * @returns moment.Moment representing the rule's 'from' property
+ */
+export const calculateFromValueWithRuleScheduleFields = (interval: string, lookback: string) => {
```

Comment:
I would propose to call this function something like - `calculateRange` or so on.

It does not return `from` property that can be used in rule straightaway, just time range in seconds. `now` is added as prefix in another function.

Reply:
Switched the function to return the now-prefixed string